### PR TITLE
Added Profiling elastic agent integration

### PR DESCRIPTION
**Pull request by iogbole**

### CODE REVIEW

**Review by christos68k — APPROVED** [review]
Left some suggestions. Also, this PR probably needs to be tagged with `backport-8.8` (not 8.1), I updated the labels, feel free to change. LGTM otherwise.

File: docs/en/observability/profiling-get-started.asciidoc [hunk 1]
```diff
@@ -124,6 +124,30 @@ The following is an example of the provided instructions for {k8s}:
 [role="screenshot"]
 image::images/profiling-k8s-hostagent.png[]
 
+[discrete]
+[[profiling-install-host-agent-elastic-agent]]
+== Install the host-agent via Elastic Agent 
+
+If you do not wish to install the host agent in standalone mode as shown in <<profiling-install-host-agent>>, you can instead allow Elastic Agent to orchestrate the installation by using the Universal Profiling Agent integration.
```
Comment (mdbirnstiehl) [inline docs/en/observability/profiling-get-started.asciidoc:131]:
```suggestion
You can also use the Universal Profiling Agent integration through the {agent} to orchestrate to install the host-agent.
```

File: docs/en/observability/profiling-get-started.asciidoc [hunk 2]
```diff
@@ -111,7 +111,7 @@ We still recommend upgrading as the latest version contains several improvements
 
 [discrete]
 [[profiling-install-host-agent]]
-== Install the host-agent
+== Install the host-agent in standalone mode
```
Comment (mdbirnstiehl) [inline docs/en/observability/profiling-get-started.asciidoc:114]: I think it's probably a good idea to tell people that there are two options before this section so they don't start with standalone and realize they want to install with the agent. I tried to do this in suggestions, but wasn't sure I was getting the formatting right, so I'll give it a try here:

```
[discrete]
[[profiling-install-host-agent]]
== Install the host-agent

You have the following options when installing the host-agent:

- <<profiling-install-host-agent-standalone>>
- <<profiling-install-host-agent-elastic-agent>>

[discrete]
[[profiling-install-host-agent-standalone]]
=== Install the host-agent in standalone mode
```

**Review by mdbirnstiehl — COMMENTED** [review]
A couple of suggestions. Let me know if something doesn't make sense.

File: docs/en/observability/profiling-get-started.asciidoc [hunk 3]
```diff
@@ -124,6 +124,30 @@ The following is an example of the provided instructions for {k8s}:
...
+If you do not wish to install the host agent in standalone mode as shown in <<profiling-install-host-agent>>, you can allow the Elastic Agent to orchestrate the installation instead by using the Universal Profiling Agent integration.
+
+To install the Universal Profiling Agent integration, complete the following steps:
+
+. Copy the `secret token` and `APM Server URL` from the Elastic Agent Integration 
++
+[role="screenshot"]
+image::images/profiling-elastic-agent.png[]
++
+. Then click on step 2 to complete the integration.
```
[the remark this answers is not in the document]
Comment (iogbole) [reply docs/en/observability/profiling-get-started.asciidoc:140]: @mdbirnstiehl  - the screenshot is included - https://github.com/elastic/observability-docs/blob/profiling-elastic-agent/docs/en/observability/profiling-get-started.asciidoc

[the remark this answers is not in the document]
Comment (mdbirnstiehl) [reply docs/en/observability/profiling-get-started.asciidoc:141]: So do we want them to click here:

image

if so, I would say "Click **Manage Universal Profiling Agent in Fleet**."

[the remark this answers is not in the document]
Comment (iogbole) [reply docs/en/observability/profiling-get-started.asciidoc]: done

File: docs/en/observability/profiling-get-started.asciidoc (hunk 2 above)
Comment (iogbole) [reply docs/en/observability/profiling-get-started.asciidoc:122]: done

**Pull request merged**